test/merge-patient-lab-iips-mocha.js

```diff
@@ -0,0 +1,147 @@
+// merge-patient-lab-iips-mocha.js
+
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.should();
+chai.use(chaiAsPromised);
+
+var _ = require('underscore');
+
+var noflo = require('noflo');
+var sinon = require('sinon');
+
+var componentFactory = require('../components/merge-patient-lab-iips');
```

Comment:
Changes to the sprint test case should be in a separate PR. This way it is clear that this PR adheres to the agreed upon behaviour.